proxy/api/src/http/source.rs

```diff
@@ -664,18 +589,29 @@ mod test {
 
         let urn = replicate_platinum(&ctx).await?;
 
-        let branch = git::Branch::local("master");
+        let branch_name = "master";
+        let revision = coco::Revision::Branch {
+            name: branch_name.to_string(),
+            peer_id: None,
+        };
+        let query = super::CommitsQuery {
+            revision: Some(revision.clone()),
+        };
         let res = request()
             .method("GET")
-            .path(&format!("/commits/{}?branch={}", urn.clone(), branch.name))
+            .path(&format!(
+                "/commits/{}?branch={}",
```

Comment:
```suggestion
                "/commits/{}?{}",
```

I’m not sure why this did not trigger an error

Reply:
It did. :D